package.json

```diff
@@ -70,7 +70,7 @@
   },
   "dependencies": {
     "@swc/helpers": "^0.3.13",
-    "@whereby/jslib-media": "whereby/jslib-media.git",
+    "@whereby/jslib-media": "whereby/jslib-media.git#holvik/socket_io_upgrade",
```

Comment:
remember to update when `jslib-media` is released

Reply:
Updated 👍